week2.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<p>A tibble data structure is: <strong><em>_</em></strong>______</p><p> </p><p>It is used for <strong><em>_</em></strong>____</p><p> </p><p><span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>